### ISSUE TITLE
Does not prompt the user for decrypt password

### ISSUE THREAD

**Issue by biharits** [TeamWin/android_device_sony_pioneer#3]
TWRP does not prompt the user for decrypt password after boot into twrp via fastboot. Didn't find any option for decryption. I want to use file manager.
Using 3.6.2

**Comment by HanabishiRecca** [comment]
For Android 12 decrypting the data is not possible atm.

**Comment by biharits** [comment]
Thanks :C

**Comment by HanabishiRecca** [comment]
There is a workaround though. You can patch the boot image to disable data encryption completely, if you don't care about it.

**Comment by biharits** [comment]
Possible to "undo"?
Tbh I need it for removing a magisk module that causing boot loop, I don't see anything in /data with adb shell. That's why I need that, otherwise I'm going to reflash my phone.

**Comment by HanabishiRecca** [comment]
> Possible to "undo"?

No. In fact it's not even possible on already encrypted partition and requires data wipe.

> Tbh I need it for removing a magisk module that causing boot loop, I don't see anything in /data with adb shell. That's why I need that, otherwise I'm going to reflash my phone.

No luck then. Thank Google for that.  
But you can use it as an opportunity to wipe data and disable encryption prior the system first boot.

**Comment by biharits** [comment]
Gonna reflash my phone then, thanks anyway.

**Comment by HanabishiRecca** [comment]
Recent TWRP 3.7.0 update finally brings support for Android 12 encryption.
https://twrp.me/site/update/2022/10/10/3.7.0-released.html

**Comment by biharits** [comment]
Hi, I tried twrp 3.7.0, unfortunately it does not prompt me to decrypt partition, do I have to do something for it?

**Comment by HanabishiRecca** [comment]
Prompt is not essential, it not show up if the device is not locked, just silently decrypts.
Are you sure data partition stll encrypted?

**Comment by biharits** [comment]
it's not entirely encrypted or idk, i can see everything in root directory ( /*) and the contents of them, but can see nothing readable folder/filename deeper
so for example:
/data/adb/-21l3$k5+234ask and such

**Comment by biharits** [comment]
how do i take screenshots in twrp?

**Comment by HanabishiRecca** [comment]
> i can see everything in root directory ( /*)

Encryption is only applied to the user data partition (`/data`).

> /data/adb/-21l3$k5+234ask and such

Yeah, you definitly have a problem.
